docs/rabbbitmq.md

```diff
@@ -0,0 +1,92 @@
+# `rabbitmq` Metrics Receiver
+
+This receiver fetches stats from a RabbitMQ node using the [RabbitMQ Management Plugin](https://www.rabbitmq.com/management.html).
+
+## Prerequisites
+
+This receiver supports RabbitMQ versions `3.8` and `3.9`.
+
+The RabbitMQ Management Plugin must be enabled by following the [official instructions](https://www.rabbitmq.com/management.html#getting-started).
+
+Also, a user with at least [monitoring](https://www.rabbitmq.com/management.html#permissions) level permissions must be used for monitoring.
+## Configuration
+
+Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your rabbitmq configuration.
+
+To configure a receiver for your rabbitmq metrics, specify the following fields:
+
+| Field                   | Required | Default                         | Description |
+| ---                     | ---      | ---                             | ---         |
+| `type`                  | required |                                 | Must be `rabbitmq`. |
+| `endpoint`              | optional | `http://localhost:15672`        | URL of node to be monitored |
+| `collection_interval`   | required |                                 | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+| `username`              | optional | required                        | The username used to connect to the server. |
+| `password`              | optional | required                        | The password used to connect to the server. |
+| `insecure`              | optional | true                            | Signals whether to use a secure TLS connection or not. If insecure is true TLS will not be enabled. |
+| `insecure_skip_verify`  | optional |                                 | Whether to skip verifying the certificate or not. A false value of insecure_skip_verify will not be used if insecure is true as the connection will not use TLS at all. |
+| `cert_file`             | optional |                             | Path to the TLS cert to use for TLS required connections. |
+| `key_file`              | optional |                             | Path to the TLS key to use for TLS required connections. |
+| `ca_file`               | optional |                             | Path to the CA cert. As a client this verifies the server certificate. If empty, uses system root CA. |
+
+Example Configuration:
```

Comment:
`s/rabbitmq_metrics/rabbitmq` and `s/rabbitmq_pipeline/rabbitmq` here as well? Similar for all the other configuration files.

Also the sample configuration should be bare minimum configuration. Can we remove the optional parameters like `endpoint` and `collection_interval`? If `username` and `password` are optional, let's remove them as well. It is ok for the second configuration sample below to be verbose.

See more context in https://github.com/GoogleCloudPlatform/ops-agent/pull/384#issue-1123802764

Reply:
Updated

Reply:
Username and password are required, that was a typo in the doc, the required was put in the wrong spot.